miqa/core/rest/project.py

```diff
@@ -47,6 +52,18 @@ def get_permissions(self, obj):
 
         return permissions
 
+    def get_artifacts(self, obj):
+        if obj.artifact_group_id is not None:
+            artifacts = Artifact.objects.filter(group__artifact__group__id=obj.artifact_group_id)
```

Comment:
```suggestion
            artifacts = obj.artifact_group.artifacts
```

(That assumes you've set a `related_name` of `'artifacts'` on the ForeignKey from Artifact to Group.)